v2/en/changelog-1015008.md

```diff
@@ -0,0 +1,148 @@
+<!---
+    @title         ChangeLog 1.15.8
+    @creator       Thibault Charbonnier
+    @created       2018-11-14 01:32 GMT
+--->
+
+# Version 1.15.8.1 - dd MMMM YYYY
+
+* upgraded the [nginx](nginx.html) core to 1.15.8.
+    * see the changes here: http://nginx.org/en/CHANGES
+* change: we now enable the GC64 mode by default in our bundled LuaJIT build for x86_64 architectures; this can be disabled with ./configure `--without-luajit-gc64`. Thanks Thibault Charbonnier for the patch.
+* feature: bundled the [lua-tablepool](https://github.com/openresty/lua-tablepool#readme) library. Thanks Thibault Charbonnier for the patch.
+* feature: ./configure: added new options `--without-stream_ssl_module` and `--without-stream`.
+* feature: ./configure: added support for the `-h` option.
+* feature: ./configure: support `@rpath` placeholder in OS X. Thanks Datong Sun for the patch.
+* feature: updated the socket_cloexec patches to support the [ngx.pipe](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/pipe.md#readme) API. Thanks spacewander for the patch.
+* bugfix: [nginx](nginx.html) did not destroy the cycle memory pool before the daemon process exits. Thanks Yuansheng Wang for the patch.
+* resty-doc indexes: skipped nginx's njs docs due to the use of special xml tags.
+* upgraded [ngx_lua](https://github.com/openresty/lua-nginx-module#readme) to 0.10.14rc4.
+    * change: we no longer support the standard Lua 5.1 interpreter (PUC-Rio Lua). It is now strongly recommended to use the OpenResty LuaJIT releases, bundled with the official OpenResty releases.
+    * change: we now print an alert when a non openresty-specific version of LuaJIT is detected since many optimizations would be missing. Thanks Thibault Charbonnier for the patch.
+    * change: we now print a warning when writing to the Lua global variable `_G`.
+    * change: we now avoid running [init_by_lua*](https://github.com/openresty/lua-nginx-module#init_by_lua) in signaller processes and when testing the [nginx](nginx.html) configuration. Thanks spacewander for the patch.
+    * change: we now avoid generating the Content-Type response header when getting all response headers. Thanks spacewander for the patch.
+    * feature: added support for ARM64 (Aarch64). Thanks Cloudflare for sponsoring this work. Thanks Dejiang Zhu and Zexuan Luo for the development work of this patch.
+    * feature: added the C API to support the [ngx.pipe](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/pipe.md#readme) API provided by [lua-resty-core](https://github.com/openresty/lua-resty-core#readme). Thanks spacewander for the patch.
+    * feature: added backlog queueing to cosocket connection pools. Thanks spacewander for the patch.
```

Comment:
Better add a link to the corresponding documentation section? This is a major feature :)

Reply:
Good point!